src/test/java/seedu/address/model/step/UniqueStepListTest.java

```diff
@@ -0,0 +1,150 @@
+package seedu.address.model.step;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.person.exceptions.DuplicatePersonException;
+import seedu.address.model.step.exceptions.DuplicateStepException;
+import seedu.address.model.step.exceptions.StepNotFoundException;
+
+
+public class UniqueStepListTest {
+
+    private static final Step TESTSTEP = new Step("Cook 1 egg."); // for the time being only
+    private static final Step TESTSTEP2 = new Step("Boil 2 egg."); // for the time being only
+    private final UniqueStepList uniqueStepList = new UniqueStepList();
+
+    @Test
+    public void contains_nullPerson_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> uniqueStepList.contains(null));
+    }
+
+    @Test
+    public void contains_stepNotInList_returnsFalse() {
+        assertFalse(uniqueStepList.contains(TESTSTEP));
+    }
+
+    @Test
+    public void contains_stepInList_returnsTrue() {
+        uniqueStepList.add(TESTSTEP);
+        assertTrue(uniqueStepList.contains(TESTSTEP));
+    }
+
+    @Test
+    public void add_nullStep_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> uniqueStepList.add(null));
+    }
+
+    @Test
+    public void add_duplicateStep_throwsDuplicatePersonException() {
+        uniqueStepList.add(TESTSTEP);
+        assertThrows(DuplicatePersonException.class, () -> uniqueStepList.add(TESTSTEP));
```

Comment:
I think it should be add_duplicateStep_throwsDuplicate**Step**Exception() and assertThrows(Duplicate**Step**Exception...)

Reply:
fixed